charts/caraml-store-registry/templates/deployment.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "caraml-store-registry.fullname" . }}
```

Comment:
Can we store the fullname in a variable since it's referenced multiple times in the template? eg:
```yaml
{{- $fullname := include "caraml-store-registry.fullname" . }}
...
metadata:
  name: {{ $fullname }}
  namespace: {{ .Release.Namespace }}
  ...
      volumes:
      - name: {{ $fullname }}-config
```